### PR TITLE
2-database schema-kavita

### PR DESCRIPTION
**Pull request by kavitappatil**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: server/cyf_future_hire.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+CREATE TABLE students 
+(
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(120) NOT NULL,
+    location VARCHAR(200) NOT NULL,
+    passing_year date NOT NULL
+);
+
+INSERT INTO students
```
Comment (Luke-Rogerson) [inline server/cyf_future_hire.sql:11]: I suggest you separate the process of creating the table(s) (the migration), with the process of seeding it with data. I would have the migration in this file, then write a little Node.js script to add bulk add data to the table.

File: server/cyf_future_hire.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
+drop table if exists students;
```
Comment (Luke-Rogerson) [inline server/cyf_future_hire.sql:1]: Do you plan to run this more than once? If so, it's going to delete your `students` table each time. Is that what you intend?

Comment (kavitappatil) [reply server/cyf_future_hire.sql:1]: it is just a first-time setup. In future, I will create more scripts for amendments.

File: server/cyf_future_hire.sql (hunk 1 above)
Comment (kavitappatil) [reply server/cyf_future_hire.sql:11]: ok, thanks

**Pull request merged**